plugins/jabra-file/fu-jabra-file-device.c

```diff
@@ -282,82 +255,53 @@ fu_jabra_file_device_file_checksum(FuJabraFileDevice *self,
 				   gboolean *match,
 				   GError **error)
 {
+	guint8 device_checksum[16] = {0x00};
+	guint8 info[] = {0x01 << 6};
+	g_autoptr(FuJabraFileCmd) cmd_req1 = fu_jabra_file_cmd_new();
+	g_autoptr(FuJabraFileCmd) cmd_req2 = fu_jabra_file_cmd_new();
+	g_autoptr(FuJabraFileCmd) cmd_rsp1 = NULL;
+	g_autoptr(FuJabraFileCmd) cmd_rsp2 = NULL;
 	g_autoptr(GString) device_checksum_str = g_string_new(NULL);
 	g_autoptr(GString) firmware_checksum_str = g_string_new(firmware_checksum);
-	guint8 device_checksum[16] = {0x00};
-	guint8 info = 0x01 << 6;
-
-	FuJabraFileTxData tx_data1 = {
-	    .txbuf =
-		{
-		    FU_JABRA_FILE_IFACE,
-		    self->address,
-		    0x00,
-		    self->sequence_number,
-		    0x47,
-		    0x03,
-		    0x03,
-		    0x00,
-		},
-	    .timeout = FU_JABRA_FILE_STANDARD_SEND_TIMEOUT,
-	};
-	FuJabraFileTxData tx_data2 = {
-	    .txbuf =
-		{
-		    FU_JABRA_FILE_IFACE,
-		    self->address,
-		    0x00,
-		    (self->sequence_number) + 1,
-		    0x47,
-		    0x03,
-		    0x03,
-		    info,
-		},
-	    .timeout = FU_JABRA_FILE_STANDARD_SEND_TIMEOUT,
-	};
-	FuJabraFileRxData rx_data = {
-	    .rxbuf = {0x00},
-	    .timeout = FU_JABRA_FILE_STANDARD_RECEIVE_TIMEOUT,
-	};
-	if (!fu_device_retry_full(FU_DEVICE(self),
-				  fu_jabra_file_device_tx_cb,
-				  FU_JABRA_FILE_MAX_RETRIES,
-				  FU_JABRA_FILE_RETRY_DELAY,
-				  (gpointer)&tx_data1,
-				  error))
+
+	/* FIXME what is this doing?? */
```

Comment:
reading the checksum on the file once it is in the device (it happens in 2 commands)